src/coreclr/jit/llvm.cpp

```diff
@@ -166,11 +202,32 @@ llvm::Type* getLlvmTypeForCorInfoType(CorInfoType corInfoType) {
     }
 }
 
+// TODO: Seems like this is not a great idea, but when looking at a sigInfo from eeGetMethodSig we have CorInfoType(s) but when looking at lclVars we have LclVarDsc or var_type(s), it would be better if a common
+// denomiator was used, but I couldn't find one so introduced this method to workaround.
```

Comment:
The rest of the Jit deals in `var_types` exclusively, and converts to that as soon as possible.

Since we're juggling 3 types of "types" here (var, CorInfo and LLVM's), I think it is fine to have these helper functions to convert between them, but we'll have to be careful about rationalizing some of the finer differences, if and when those come up.